packages/@aws-cdk/aws-msk-alpha/lib/cluster.ts

```diff
@@ -103,6 +103,13 @@ export interface ClusterProps {
    * @default - 1000 GiB EBS volume
    */
   readonly ebsStorageInfo?: EbsStorageInfo;
+  /**
```

Comment:
```suggestion
  readonly ebsStorageInfo?: EbsStorageInfo;
  
  /**
```

Reply:
Hey @kaizencc,

This change has broken the build, as it looks to have added a space character. 

Seeing the logs [here](https://[ops-cdk-autobuild-prs-autobuildv2github-buildlogs-g9jsj1nfmnjp.s3.amazonaws.com/29bcc86b-2069-44b4-9721-2154252c23fc/build.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=ASIAWYORDRBKPF6TTVLV%2F20231119%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20231119T040926Z&X-Amz-Expires=600&X-Amz-SignedHeaders=host&X-Amz-Security-Token=IQoJb3JpZ2luX2VjEIX%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FwEaCXVzLWVhc3QtMSJHMEUCIDNvsxhMbZZlL1x1Ft9BBE%2Fq82wzDTIX9l5Cz7c9WDzbAiEA%2B4E1BiY6mI0k%2BbFkltjZx7E7jowYsBJVttZkGT%2F2lVsq3AMIzf%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FARACGgw0NjQ4MzE4NzUxNTYiDCa4iF3QChu7MVR%2BkCqwA1qjnIpXHc3uuSgxT74zYxlvab%2BNQ8vz1pKvtToYYoIpgaxsuhfVNibnoHaybyI1QwUt922iiDv0yaEJfvL5MlhQilviySBpPIULzC40dx4MwL58y9f4RiWQElNQDhKwylfYUhsWfJA973j6ts6rJHNHMNCljvCBBfUTs3gEMmRMSlDLLorcMTR7Apts3kbdncY43yk4CqCuwVKz4WNHykhxq2jqGWD1xOGxS4Vt4QxXWJDniR2wJ0zIHsNzV0Rl8HFRlmhbxesE1AkHc2eb8GRhr9q4ZQ2a1poEKM4OqL%2BNFfkXXYNeZwK3f3epeYhTB73BZkhN7BOlOTWO6n1DOKn7C4Pgrjx5f96djLRYu3heb3%2FIZjq4TDqFvBn2aC6wfhgWjkJop4IcILoJdIS8LsvcTiOeF0AGiTuHyiFnJlDr3F3mMrbTmOkMYxnau0RyLOKLgemN1yWMxMRxxgTlCzqbSXDkkb8An67athDv8M2cne%2F3KJDh7IHPpmhJv7pV7rw%2FJY9BS1p6lAe4p8PcaxYpNBaL3vkeHq5ZRcJ2bsGnSu6h8QT39T7gW6pnp0cgxjDylOaqBjqeAWr7NXFPF0GcVVT1gLaFj8oeVdCU05pyWi9WIgfF4CIgmz%2B1%2BDGpLKdwZ%2BTjGI2i5yp5PHAQ8FnBpym2vY1gBZ3otKz9PeHfuEvaSdxtQ5m%2FUkHP3BUZ1lx6vUn%2BFOM33AvQpUa%2Fqs1kEA17n0fJyPVPyWgpzcsDEoBkKyV7z%2F2OBNtmj2bORNJNT4A51VI4cB1hCI1eK0B5Ac5Ah50E&X-Amz-Signature=6fc311b3a82415f82ccdccbbe0e818a2764adfb673258ea3e8d8c5679b967471](https://ops-cdk-autobuild-prs-autobuildv2github-buildlogs-g9jsj1nfmnjp.s3.amazonaws.com/29bcc86b-2069-44b4-9721-2154252c23fc/build.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=ASIAWYORDRBKPF6TTVLV%2F20231119%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20231119T040926Z&X-Amz-Expires=600&X-Amz-SignedHeaders=host&X-Amz-Security-Token=IQoJb3JpZ2luX2VjEIX%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FwEaCXVzLWVhc3QtMSJHMEUCIDNvsxhMbZZlL1x1Ft9BBE%2Fq82wzDTIX9l5Cz7c9WDzbAiEA%2B4E1BiY6mI0k%2BbFkltjZx7E7jowYsBJVttZkGT%2F2lVsq3AMIzf%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FARACGgw0NjQ4MzE4NzUxNTYiDCa4iF3QChu7MVR%2BkCqwA1qjnIpXHc3uuSgxT74zYxlvab%2BNQ8vz1pKvtToYYoIpgaxsuhfVNibnoHaybyI1QwUt922iiDv0yaEJfvL5MlhQilviySBpPIULzC40dx4MwL58y9f4RiWQElNQDhKwylfYUhsWfJA973j6ts6rJHNHMNCljvCBBfUTs3gEMmRMSlDLLorcMTR7Apts3kbdncY43yk4CqCuwVKz4WNHykhxq2jqGWD1xOGxS4Vt4QxXWJDniR2wJ0zIHsNzV0Rl8HFRlmhbxesE1AkHc2eb8GRhr9q4ZQ2a1poEKM4OqL%2BNFfkXXYNeZwK3f3epeYhTB73BZkhN7BOlOTWO6n1DOKn7C4Pgrjx5f96djLRYu3heb3%2FIZjq4TDqFvBn2aC6wfhgWjkJop4IcILoJdIS8LsvcTiOeF0AGiTuHyiFnJlDr3F3mMrbTmOkMYxnau0RyLOKLgemN1yWMxMRxxgTlCzqbSXDkkb8An67athDv8M2cne%2F3KJDh7IHPpmhJv7pV7rw%2FJY9BS1p6lAe4p8PcaxYpNBaL3vkeHq5ZRcJ2bsGnSu6h8QT39T7gW6pnp0cgxjDylOaqBjqeAWr7NXFPF0GcVVT1gLaFj8oeVdCU05pyWi9WIgfF4CIgmz%2B1%2BDGpLKdwZ%2BTjGI2i5yp5PHAQ8FnBpym2vY1gBZ3otKz9PeHfuEvaSdxtQ5m%2FUkHP3BUZ1lx6vUn%2BFOM33AvQpUa%2Fqs1kEA17n0fJyPVPyWgpzcsDEoBkKyV7z%2F2OBNtmj2bORNJNT4A51VI4cB1hCI1eK0B5Ac5Ah50E&X-Amz-Signature=6fc311b3a82415f82ccdccbbe0e818a2764adfb673258ea3e8d8c5679b967471)), the following is the only error seen:
```shell
@aws-cdk/aws-msk-alpha: /codebuild/output/src3647605821/src/github.com/aws/aws-cdk/packages/@aws-cdk/aws-msk-alpha/lib/cluster.ts
@aws-cdk/aws-msk-alpha:   106:1  error  Trailing spaces not allowed  no-trailing-spaces
@aws-cdk/aws-msk-alpha: âœ– 1 problem (1 error, 0 warnings)
@aws-cdk/aws-msk-alpha:   1 error and 0 warnings potentially fixable with the `--fix` option.
```

This change actually looks like a typo from my eyes, as the additional newline appears not appropriate for the existing code style? Please correct me if I am wrong.

Reply:
Not a typo. The code style of this package is wrong and not inline with the rest of cdk. The issue was that there was a hidden tab that the build didn't like.